test/DnsClient.Tests/LookupTest.cs

```diff
@@ -765,6 +766,27 @@ public async Task Lookup_Query_NaPtr()
             Assert.NotNull(host.HostName);
         }
 
+        [Fact]
+        public async Task Lookup_Query_CERT()
+        {
+            var client = new LookupClient(NameServer.Cloudflare);
+            var result = await client.QueryAsync("d1.domain1.dcdt31.healthit.gov", QueryType.CERT).ConfigureAwait(false);
```

Comment:
Just a question: Is this record going to change or disappear one day? If so, this test might become flaky over time -.-
If the record is stable, the test is fine, otherwise, we should probably remove it.

Reply:
ok that's good enough for now then ;) thanks